wnt/wntclientapi.md

```diff
@@ -0,0 +1,2502 @@
+|                                   | -   Updated *protocol version*    |
+|                                   |     to 3.                         |
+|                                   |                                   |
+|                                   | -   Added note about              |
+|                                   |     *analytics\_packet*           |
+|                                   |     measurement change.           |
+|                                   |                                   |
+|                                   | -   Added *Get components         |
+|                                   |     information* and *Get         |
+|                                   |     scratchpad status* methods.   |
+|                                   |                                   |
+|                                   | -   Added floor plan image size   |
+|                                   |     to *Create* and *Update floor |
+|                                   |     plan* messages.               |
+|                                   |                                   |
+|                                   | -   Changed *Create* and *Update  |
+|                                   |     floor plan* messages to       |
+|                                   |     return all floor plan fields. |
+|                                   |                                   |
+|                                   | -   Added coordinate conversion   |
+|                                   |     related data to *Create* and  |
+|                                   |     *Update floor plan* messages. |
+|                                   |                                   |
+|                                   | -   Changed *network* *id* to     |
+|                                   |     number in *Create*, *Update   |
+|                                   |     and Delete network*.          |
+|                                   |                                   |
+|                                   | -   Removed *pixel\_location\_x*  |
+|                                   |     and *pixel\_location\_y* from |
+|                                   |     *Set node metadata* message.  |
+|                                   |                                   |
+|                                   | -   Added *remote\_api\_response* |
+|                                   |     to Influx measurements.       |
+|                                   |                                   |
+|                                   | -   Split *analytics\_packet*     |
+|                                   |     Influx measurement's          |
+|                                   |     *travel\_time\_ms* column to  |
+|                                   |     *travel\_time\_ms\_qos0* and  |
+|                                   |     *travel\_time\_ms\_qos1*.     |
++-----------------------------------+-----------------------------------+
```

Comment:
This part should be always visible as it is easy reference for what has changed in the API